libr/util/print.c

```diff
@@ -1629,6 +1631,26 @@ static bool issymbol(char c) {
 	}
 }
 
+static bool check_arg_name (char *p, RList *vars) {
```

Comment:
Can u explain me why do we need this function? (i couldn't understand that's why asking)

Reply:
So this function gets called from the r_colorize_opcode() loop when an arg of disasmed instruction is encountered. It returns true if it is equal to the name of one of the local vars.
I've added it to not clutter too much the r_colorize_opcode loop, but it is not necessary at all